aid.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) Facebook, Inc., 2018
+ *
+ *  Copyright holder grants permission for redistribution and use in source
+ *  and binary forms, with or without modification, provided that the
+ *  following conditions are met:
+ *     1. Redistribution of source code must retain the above copyright
+ *        notice, this list of conditions, and the following disclaimer
+ *        in all source files.
+ *     2. Redistribution in binary form must retain the above copyright
+ *        notice, this list of conditions, and the following disclaimer
+ *        in the documentation and/or other materials provided with the
+ *        distribution.
+ *
+ *  "DISCLAIMER OF LIABILITY
+ *
+ *  THIS SOFTWARE IS PROVIDED BY DAN HARKINS ``AS IS'' AND
```

Comment:
I don't think we want this particular clause on this source file as Dan Harkins didn't have anything to do with it.  Maybe "the copyright holders" as per https://opensource.org/licenses/BSD-2-Clause ?

Reply:
Oh, copy/paste…